components/asana/sources/new-workspace/new-workspace.mjs

```diff
@@ -1,9 +1,10 @@
-const asana = require("../../asana.app.js");
+import asana from "../../asana.app.mjs";
 
-module.exports = {
-  name: "Workspace Added",
+export default {
+  type: "source",
   key: "asana-new-workspace",
-  description: "Emits an event each time you add a new workspace/organization.",
+  name: "New Workspace Added",
+  description: "Emit new event each time you add a new workspace/organization.",
   version: "0.0.1",
```

Comment:
From this previous comment - https://github.com/PipedreamHQ/pipedream/pull/2487#discussion_r842918832
```suggestion
  version: "0.1.0",
```